legend.go

```diff
@@ -83,12 +83,12 @@ func makeLegend() (Legend, error) {
 // draw draws the legend to the given draw.Canvas.
 func (l *Legend) draw(c draw.Canvas) {
 	iconx := c.Min.X
-	textx := iconx + l.ThumbnailWidth + l.TextStyle.Width(" ")
-	xalign := 0.0
+	sty := l.TextStyle
+	textx := iconx + l.ThumbnailWidth + sty.Width(" ")
 	if !l.Left {
 		iconx = c.Max.X - l.ThumbnailWidth
 		textx = iconx - l.TextStyle.Width(" ")
-		xalign = -1
+		sty.XAlign -= 1.
```

Comment:
If it's `=-`, then any alignment that was already there gets overwritten. I changed it to `-=` so that any existing alignment is adjusted rather than ignored.

The linter suggested `sty.XAlign--`, so I changed it to that.